src/main/java/nextstep/subway/path/domain/Path.java

```diff
@@ -7,18 +7,30 @@
 
 public class Path {
     private final List<Station> stations;
-    private final long distance;
+    private final int distance;
+    private int fare = 0;
 
     public Path(List<Station> stations, double distance) {
         this.stations = stations;
-        this.distance = (long) distance;
+        this.distance = (int) distance;
+
+        PathFare pathFare = PathFare.match(this.distance);
+        this.fare = pathFare.getFare(this.distance);
+    }
+
+    public void calculateFare(int age, int lineFare) {
+        fare = FareCalculator.calculate(age, fare, lineFare);

```

Comment:
1. 노선에 따른 가격정책 반영
2. 나이에 따른 가격정책 반영

이 순서대로 적용되는게 코드에 드러나면 좋을것 같아요! 🤔
```suggestion
    public void calculateFare(MemberDiscountPolicy memberDiscountPolicy, LineFarePolicy lineFarePolicy) {
        fare = lineFarePolicy.getFare(fare);
        fare = memberDiscountPolicy.getFare(fare);

```
이런 흐름은 어떨까요? 😀

Reply:
넵 코드에서 반영되는 흐름이 잘 보이도록 수정해보겠습니다!